docs/user/README.md

```diff
@@ -269,31 +269,67 @@ variables is provided here. If an environment variable is not set, the specified
 | MODELARDBD_UNCOMPRESSED_DATA_BUFFER_CAPACITY     | 65536       | The capacity of each uncompressed data buffer as the number of elements in the buffer where each element is a timestamp and a value. Note that the resulting size of the buffer has to be a multiple of 64 bytes to avoid the actual capacity being larger than the requested due to internal alignment. |
 
 ## Docker
-An environment that includes a local [MinIO](https://min.io/) instance and an edge node using the [MinIO](https://min.io/)
-instance as the remote object store, can be set up using [Docker](https://docs.docker.com/). Note that since
-[Rust](https://www.rust-lang.org/) is a compiled language and a more dynamic `modelardbd` configuration might be needed,
-it is not recommended to use the [Docker](https://docs.docker.com/) environment during active development of `modelardbd`.
-It is however ideal to use for experimenting with `modelardbd` or when developing components that utilize `modelardbd`.
-
-Downloading [Docker Desktop](https://docs.docker.com/desktop/) is recommended to make maintenance of the created
-containers easier. Once [Docker](https://docs.docker.com/) is set up, the [MinIO](https://min.io/) instance can be
-created by running the services defined in [docker-compose-minio.yml](/docker-compose-minio.yml). The services can
-be built and started using the command:
+Two different [Docker](https://docs.docker.com/) environments are included to make it easy to experiment with the 
+different use cases of ModelarDB. The first environment sets up a single instance of `modelardbd` that only uses local 
+storage. Time series data can be ingested into this instance, compressed, and saved to local storage. The compressed 
+data in local storage can then be queried. The second environment covers the more complex use case of ModelarDB where 
+multiple instances of `modelardbd` are deployed in a cluster with a manager that is responsible for managing the 
+cluster. A single edge node and a single cloud node is set up in the cluster. Data can be ingested into the edge or 
+cloud node, compressed, and transferred to a remote object store. The compressed data in the remote object store can 
+then be queried through the cloud node or by directing the query through the manager node.
```

Comment:
Maybe change the description of query processing in the cloud to first say that users can submit the query to the manager, which through workload balancing, assigns the query to a cloud node for execution. As a side note at the end we can write that users also can select a specific cloud node manually, preferably with an example of why they may want to do so, but that querying through the manager for workload balancing is recommended.

Reply:
I prefer to keep the descriptions very brief in this Docker section and talk about it more detail in the Usage documentation instead (which was updated in the other PR). I will look at those sections again to make sure this is clear. 

This text is also a description of what can be done in the environment that is set up which only includes a single cloud node so talking about workload balancing here does not make much sense,.